tests/test_datasets.py

```diff
@@ -7,16 +7,24 @@ class TestDatasets(unittest.TestCase):
     # Bad URL tests
     def test_nasa_battery_bad_url_download(self):
         from prog_models.datasets import nasa_battery
-        BAD_URL = "BADURLTEST"
-        nasa_battery.urls = {'RW1':"https://"+BAD_URL}
+        nasa_battery.urls = {'RW1':"https://BADURLTEST"}
         with self.assertRaises(ConnectionError):
             (desc, data) = nasa_battery.load_data(1)
+        # Legit website, but it's not the repos
+        nasa_battery.urls = {'RW1':"https://github.com/nasa/prog_models"}
+        with self.assertRaises(ConnectionError):
+            (desc, data) = nasa_battery.load_data(1)
+
     def test_nasa_cmapss_bad_url_download(self):
         from prog_models.datasets import nasa_cmapss
         BAD_URL = "https://"+"BADURLTEST"
         nasa_cmapss.URL = BAD_URL
         with self.assertRaises(ConnectionError):
             (train, test, results) = nasa_cmapss.load_data(1)
+        # Legit website, but it's not the repos
+        nasa_cmapss.URL = "https://github.com/nasa/prog_models"
+        with self.assertRaises(ConnectionError):
+            (desc, data) = nasa_cmapss.load_data(1)
```

Comment:
Are you not expecting a tuple from cmapps like in line 23?

Reply:
```suggestion
            (train, test, results) = nasa_cmapss.load_data(1)
```
good catch!